### ISSUE TITLE
Supports API endpoints?

### ISSUE THREAD

**Issue by mrlife** [aspnet/AzureSignalR-samples#158]
With Azure SignalR Service being the [recommended hosting model](https://docs.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/server?view=aspnetcore-5.0#azure-signalr-service) for Blazor Server apps, does it support everything a Blazor Server app supports, such as API endpoints?

**Comment by JialinXin** [comment]
@mrlife Would you explain a little more? What feature are you looking for support?

**Comment by mrlife** [comment]
@JialinXin Thank you, does this help? https://github.com/dotnet/aspnetcore/issues/30204

**Comment by JialinXin** [comment]
This seems nothing special for Azure SignalR. You can have a try.

**Comment by mrlife** [comment]
@JialinXin Thank you, are you saying this is supported?

**Comment by JialinXin** [comment]
Yes. I think your requirement is in Blazor server side which should be supported naturally. Please try and let us know if you meet any problem.
